### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by sumitk-singh**
## created the home page for quiz app

### CODE REVIEW

File: src/components/Header/Header.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+.brand-logo-img {
+    width: 6.25rem;
+    height: 3.125rem;
+}
+
+.nav-bar-icons-list,
+.nav-bar-links-list {
+    display: flex;
+    gap: 15px;
```
Comment (devAmlan) [inline src/components/Header/Header.css:26]: you can use rem here

File: src/Pages/home-page/Home.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    gap: 2rem;
+    justify-content: center;
+   
+}
+
+#btn-Quiz-Category{
+    
+    height: 12rem;
+    width: 18rem;
+    border-radius: 15px;
```
Comment (devAmlan) [inline src/Pages/home-page/Home.css:54]: you can use rem here

**Review by devAmlan — APPROVED** [review]
The UI is awesome and the Code is clean. Suggested some changes, please check. Apart from that everything looks good and ready to merge

File: src/components/Header/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                            <Link to="/"><img  className="brand-logo-img" src={logo} alt="brand-logo" /></Link>
+                        </div>
+                        <div className="nav-bar-links">
+                            <ul className="nav-bar-links-list">
+                               
+                            </ul>
+                        </div>
+                        <div className="nav-bar-icons">
+                            <ul className="nav-bar-icons-list">
+                                <li>Login/ Signup</li>
```
Comment (humzahasan) [inline src/components/Header/Header.jsx:20]: I guess you will be changing this into two separate li.

**Review by humzahasan — APPROVED** [review]
Code changes looks good. 

Good to merge!

**Comment by Ranjeetk489** [comment]
image
text size mismatch needs to be corrected

image
the logo feels squeezed

The login/signup can use bit more styling, Aside that clean UI and good to merge

File: src/components/Header/Header.jsx (hunk 3 above)
Comment (sumitk-singh) [reply src/components/Header/Header.jsx:20]: no it's single

**Pull request merged**